strings/src/main/res/values/strings.xml

```diff
@@ -996,6 +996,8 @@
     <string name="delete_project_confirm_message">Are you sure you want to delete the current project?</string>
     <string name="delete_project_yes">Yes</string>
     <string name="delete_project_no">No</string>
+    <string name="cannot_delete_project_title">Cannot delete the current project!</string>
+    <string name="cannot_delete_project_message">You have unsent instances. Please send them or delete first.</string>
```

Comment:
I'm thinking we probably want to call these "filled" or "saved" forms? I also don't think we should have the "!" on the title, and we can probably drop "current". cc/ @lognaturel.

Reply:
I updated those strings so that now they look more like those from https://www.figma.com/file/nDsMAYQDIflyl7rznEhwnN/ODK-Collect-Multiple-Project-Support-(v2021.2)?node-id=0%3A1